apps/editor.planx.uk/src/components/EditorNavMenu/styles.tsx

```diff
@@ -45,30 +45,46 @@ export const MenuButton = styled(IconButton, {
 })<{ isActive: boolean }>(({ theme, isActive, disabled }) => ({
   color: theme.palette.text.primary,
   width: "100%",
-  border: "1px solid transparent",
   justifyContent: "flex-start",
+  gap: theme.spacing(0.65),
   alignItems: "flex-start",
   borderRadius: "3px",
   "&:hover": {
     background: theme.palette.common.white,
-    borderColor: theme.palette.border.light,
+    boxShadow: "0 1px 1.5px 0 rgba(0, 0, 0, 0.2)",
   },
   ...(isActive && {
     background: theme.palette.common.white,
     color: theme.palette.text.primary,
-    border: `1px solid ${theme.palette.border.main}`,
-    "&:hover": {
-      borderColor: theme.palette.border.main,
-    },
+    boxShadow: "0 1px 1.5px 0 rgba(0, 0, 0, 0.2)",
```

Comment:
This `isActive` condition doesn't work for both flows and team settings.

<img width="727" height="433" alt="Image" src="https://github.com/user-attachments/assets/bbdca26e-99c0-42b6-a02f-b797e48fdc6e" />

<img width="722" height="687" alt="Image" src="https://github.com/user-attachments/assets/ab71ec4f-e618-45d3-ac46-40e48ba118e4" />